Sources/ZIPFoundation/Data+Compression.swift

```diff
@@ -10,6 +10,10 @@
 
 import Foundation
 
+#if os(macOS) || os(iOS) || os(watchOS) || os(tvOS)
+import zlib
+#endif
```

Comment:
`#if canImport(zlib)`?

Reply:
Only reason I didn't do that was because I couldn't test on all platforms that might support zlib, so I wanted to be conservative. But if folks would prefer to go broad and include all platforms with zlib, then I'd be more than happy to change my PR